src/System/Analytics/trackingMiddleware.ts

```diff
@@ -16,6 +17,8 @@ interface TrackingMiddlewareOptions {
 }
 
 export function trackingMiddleware(options: TrackingMiddlewareOptions = {}) {
+  const trackPageview = debounce(window.analytics?.page!, 30)
```

Comment:
So — what about this broke tracking @damassi

Reply:
No longer tracked page on transition between SPA pages